.github/workflows/test.yml

```diff
@@ -9,14 +9,21 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: ${{ fromJSON(vars.DEFAULT_JOB_TIMEOUT_MINUTES) }}
     steps:
-    - uses: actions/checkout@v4
-    - name: Use Node.js
-      uses: actions/setup-node@v4
-      with:
-        node-version: '18.x'
-    - name: Build
-      run: |
+      - uses: actions/checkout@v4
```

Comment:
Should the indentation remain as it was? I can never remember which is preferred, but yaml.org shows it the other way.

Reply:
my ide likes it the other way for some reason but we go with he official way